### PR TITLE
docs: update slack info in readme

### PR DESCRIPTION
**Pull request by IronCore864**
## Description

Update slack info.

### CODE REVIEW

File: docs/README_zh.md [hunk 1]
```diff
@@ -124,8 +124,18 @@ DevStream已经支持许多工具，而且还在不断增加。关于支持的
 
 我们将定期组织 "DevStream Community Meeting"，请访问 [WIKI](https://github.com/devstream-io/devstream/wiki) 页面了解详情。
 
-- 在 <a href="https://cloud-native.slack.com/archives/C03LA2B8K0A" target="_blank">Slack</a> 给我们留言。
-- 对于中国用户，微信群的二维码如下：
+Please join our Slack channel. Here's how:
```
Comment (aFlyBird0) [inline docs/README_zh.md:127]: 这句忘翻译了，不过无伤大雅以后再说。

**Review by aFlyBird0 — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -121,8 +121,18 @@ First of all, thanks for wanting to contribute to DevStream! For more details on
 
 We will regularly organize `DevStream Community Meeting`, please visit the [wiki](https://github.com/devstream-io/devstream/wiki) page for details.
 
-- Message us on <a href="https://cloud-native.slack.com/archives/C03LA2B8K0A" target="_blank">Slack.</a>
-- For Chinese users, the WeChat group QR code is as below:
+Please join our Slack channel. Here's how:
+
+1. [Invite yourself to CNCF's Slack if you haven't done so](https://slack.cncf.io). 
+    - Input your email address, and click "get my invite."
```
Comment (daniel-hutao) [inline README.md:127]: Should we change the "get my invite." to "get my invite". ?

Same as below.

**Comment by aFlyBird0** [comment]
It seems that you have to resolve conflicts. Because:

image

File: docs/README_zh.md [hunk 3]
```diff
@@ -124,8 +124,18 @@ DevStream已经支持许多工具，而且还在不断增加。关于支持的
...
+
+1. [给自己发送邀请进入CNCF的Slack](https://slack.cncf.io). 
+    - 输入邮箱地址，点击"get my invite"。
+    - 打开收件箱，找到邀请邮件，点击邮件中的"join now".
```
Comment (daniel-hutao) [inline docs/README_zh.md:131]: 句号混用了“。”和“.”

**Review by daniel-hutao — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (IronCore864) [reply README.md:127]: Amerian English recommends putting the trailing punctuation within the closing quote. I recommend everyone who wants to contribute to English documentation have the "Grammarly" browser plugin as well as [this book](https://www.amazon.com/Dreyers-English-Utterly-Correct-Clarity/dp/0812995708).

File: README_zh.md [hunk 4]
```diff
@@ -124,8 +124,18 @@ DevStream已经支持许多工具，而且还在不断增加。关于支持的
 
 我们将定期组织 "DevStream Community Meeting"，请访问 [WIKI](https://github.com/devstream-io/devstream/wiki) 页面了解详情。
 
-- 在 <a href="https://cloud-native.slack.com/archives/C03LA2B8K0A" target="_blank">Slack</a> 给我们留言。
-- 对于中国用户，微信群的二维码如下：
+另外，请加入我们的Slack channel，请参见如下步骤：
+
+1. [给自己发送进入CNCF的Slack的邀请](https://slack.cncf.io).
```
Comment (daniel-hutao) [inline README_zh.md:129]: . -> 。 or . -> ： here？

**Review by daniel-hutao — CHANGES_REQUESTED** [review]

**Review by aFlyBird0 — APPROVED** [review]

**Review by daniel-hutao — APPROVED** [review]

**Pull request merged**